client/blocks/product-purchase-features/product-purchase-features-list/jetpack-search.jsx

```diff
@@ -0,0 +1,29 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import PurchaseDetail from 'components/purchase-detail';
+
+export default localize( ( { selectedSite, translate } ) => {
+	return (
+		<div className="product-purchase-features-list__item">
+			<PurchaseDetail
+				icon="search"
+				title={ translate( 'Search (Beta)' ) }
+				description={ translate(
+					'Replace the default WordPress search with better results and filtering that will help your users find what they are looking for.'
```

Comment:
I think this line is causing the linter to fail because it's too long. We should be able to use concatenation here to get the line length down so that we're pleasing the linter.

```js
'Replace the default WordPress search with better results ' +
'and filtering that will help your users find what they are looking for.'
```

Reply:
Does this work ok for pulling out translations? I was assuming that breaking up the string might break things.

I think the problem here is the linting rules :)

Reply:
It's listed as a good example here:

https://github.com/Automattic/i18n-calypso#strings-only